### PR TITLE
feat(webpaas): update code for new catalog

### PR DESCRIPTION
**Pull request by varun257**
| Question         | Answer
| ---------------- | ---
| Branch?          | `feat/web-paas-prod`
| Bug fix?         | yes
| New feature?     | no
| Breaking change? | yes
| Tickets          | Fix #MANAGER-5671
| License          | BSD 3-Clause

## Description

fix new catalog issue
fix qa issues

### CODE REVIEW

**Review by frenautvh — APPROVED** [review]

File: packages/manager/modules/web-paas/src/add/add.controller.js [hunk 1]
```diff
@@ -126,10 +139,7 @@ export default class WebPassAddCtrl {
 
   scrollToTop() {
     this.$timeout(() => {
-      document.getElementById('web-pass-add-header').scrollIntoView({
-        behavior: 'smooth',
-      });
-      document.getElementById('web-pass-add-alert').focus();
+      document.getElementById(this.PAGE_SECTION_HEADER).scrollIntoView(false);
```
Comment (radireddy) [inline packages/manager/modules/web-paas/src/add/add.controller.js:142]: use $anchorScroll
scrollToTop() {
this.$location.hash(this.PAGE_SECTIONS.BACK_BUTTON);
this.$anchorScroll();
}

File: packages/manager/modules/web-paas/src/components/additional-option/controller.js [hunk 2]
```diff
@@ -1,23 +1,31 @@
...
+        ((DEFAULT_ENVIRONMENT + (stagingQuantity || 1)) *
+          price.value *
+          this.option.quantity) /
+        STORAGE_MULTIPLE
+      );
     }
-    return `${price.value * (this.option.quantity || 0)} ${
-      this.user.currency.symbol
-    }`;
+    return price.value * (this.option.quantity || 0);
```
Comment (radireddy) [inline packages/manager/modules/web-paas/src/components/additional-option/controller.js:28]: if you set quantity to zero by default then you dont need OR option

File: packages/manager/modules/web-paas/src/details/service/change-offer/change-offer.controller.js [hunk 3]
```diff
@@ -59,10 +60,7 @@ export default class {
 
   scrollToTop() {
     this.$timeout(() => {
-      document.getElementById('web-pass-add-header').scrollIntoView({
-        behavior: 'smooth',
-      });
-      document.getElementById('web-pass-add-alert').focus();
+      document.getElementById('web-pass-add-header').scrollIntoView(false);
```
Comment (radireddy) [inline packages/manager/modules/web-paas/src/details/service/change-offer/change-offer.controller.js:63]: scrollToTop() {
this.$location.hash(this.PAGE_SECTIONS.BACK_BUTTON);
this.$anchorScroll();
}

File: packages/manager/modules/web-paas/src/translations/Messages_fr_FR.json [hunk 4]
```diff
@@ -31,6 +31,6 @@
   "web_paas_storage_description": "{{ storage }} Go",
   "web_paas_manage_user_licences": "Gérer mes licences utilisateur",
   "web_paas_change_range": "Changer d’offre",
-  "web_paas_region_GRA7": "Gravelines",
+  "web_paas_region_GRA7": "Gravelines (GRA7)",
```
Comment (radireddy) [inline packages/manager/modules/web-paas/src/translations/Messages_fr_FR.json:34]: why is this hardcoded?

File: packages/manager/modules/web-paas/src/add/add.controller.js (hunk 1 above)
Comment (varun257) [reply packages/manager/modules/web-paas/src/add/add.controller.js:142]: Tried `AnchorScroll` but faced the same issue. This resolved the issue

**Review by radireddy — APPROVED** [review]

**Pull request merged**